test/e2e/util/fixture.go

```diff
@@ -38,8 +38,6 @@ func createClusterOpts(ctx context.Context, client crclient.Client, hc *hyperv1.
 	switch hc.Spec.Platform.Type {
 	case hyperv1.AWSPlatform:
 		opts.InfraID = hc.Name
-	case hyperv1.KubevirtPlatform:
-		opts.KubevirtPlatform.RootVolumeSize = 16
```

Comment:
while investigating the NTO failure, i hit a diskpressure issue. This was caused by the test driver not observing the requested root volume size of 32Gi due to a hard coded override. i've removed this override in this PR to help with reliability moving forward.